src/Popover/PopoverAnimationDefault.js

```diff
@@ -36,7 +36,9 @@ function getStyles(props, context, state) {
   };
 }
 
-class PopoverDefaultAnimation extends Component {
+class PopoverAnimationDefault extends Component {
```

Comment:
Reverted and moved this change to https://github.com/callemall/material-ui/pull/4955